tests/src/test/scala/whisk/core/containerpool/kubernetes/test/KubernetesClientTests.scala

```diff
@@ -121,7 +121,7 @@ class KubernetesClientTests
     implicit val kubernetes = new TestKubernetesClient
     val id = ContainerId("id")
     val container = new KubernetesContainer(id, ContainerAddress("ip"), "127.0.0.1", "docker://foo")
-    container.suspend()
+    Await.result(container.suspend(), 500.milliseconds)
```

Comment:
Please use the `await` helper defined on the top of this file. It normalizes timeouts throughout.